Cargo.toml

```diff
@@ -11,11 +11,11 @@ rust-version = "1.79.0"
 
 [workspace.dependencies.soroban-env-host-prev]
 package = "soroban-env-host"
-version = "=21.2.1"
+version = "=21.2.2"
```

Comment:
CI seems to be complaining about this change; there is really no need for it as we no longer need to simulate protocol 21. So maybe just revert it and hopefully make CI pass?

Reply:
that's what I get for trying to get the latest version